src/usr/local/www/status_ipsec.php

```diff
@@ -339,7 +342,11 @@ function print_ipsec_body() {
 					print("</td>\n");
 					print("<td>\n");
 
-					print(htmlspecialchars($childsa['encr-alg']) . '<br/>');
+					print(htmlspecialchars($childsa['encr-alg']));
+					if ($childsa['encr-alg'] != '3DES_CBC') {
+						print('_' . htmlspecialchars($childsa['encr-keysize']));
+					}
+                                        print('<br/>');
```

Comment:
This line is still using spaces

Reply:
it's already in the last snapshot:
php`if (!empty($childsa['encr-keysize'])) {
          print(" (" . htmlspecialchars($childsa['encr-keysize']) . ")");
}
print('<br/>');`

PR can be closed